tests/acceptance/shared/util/constants.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import "flag"
+
+// global configurations and flags
+var (
+	Arch             = flag.String("arch", "amd64", "a string")
+	Destroy          = flag.Bool("destroy", false, "a bool")
+	KubeConfigFile   string
```

Comment:
Instead of defining a bunch of independent global varibales and then assigning them in `BuildCluster` You should define a Cluster struct, that is fillout and returned in `BuildCluster`. That would greatly simplify this package design.